### PR TITLE
feat: Add optional credentials_file_path input for setup-gcloud (#153)

### PR DESCRIPTION
**Pull request by averikitsch**
PR #153 

UNIT
https://github.com/GoogleCloudPlatform/github-actions/runs/1300545920?check_suite_focus=true
https://github.com/GoogleCloudPlatform/github-actions/actions/runs/325114551
IT
https://github.com/GoogleCloudPlatform/github-actions/runs/1300545948?check_suite_focus=true

### CODE REVIEW

File: setup-gcloud/README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ steps:
     project_id: ${{ secrets.GCP_PROJECT_ID }}
     service_account_key: ${{ secrets.GCP_SA_KEY }}
     export_default_credentials: true
+    credentials_file_path: '/tmp/gcloud-credentials'
```
Comment (bharathkkb) [inline setup-gcloud/README.md:66]: nit: I think it's better if we do not show this for our example invocation. The standard behavior should be not setting this.

File: setup-gcloud/dist/index.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ module.exports =
 /******/ 	// the startup function
 /******/ 	function startup() {
 /******/ 		// Load entry module and return exports
-/******/ 		return __webpack_require__(738);
+/******/ 		return __webpack_require__(325);
```
Comment (bharathkkb) [inline setup-gcloud/dist/index.js:37]: nit: remove dist

File: setup-gcloud/README.md (hunk 1 above)
Comment (bharathkkb) [reply setup-gcloud/README.md:66]: Perhaps we can showcase this if needed as an alternate config like here?
https://github.com/GoogleCloudPlatform/github-actions/tree/master/get-secretmanager-secrets#authorization

File: setup-gcloud/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { run } from './setup-gcloud';
```
Comment (bharathkkb) [inline setup-gcloud/src/index.ts:1]: nit: header

File: setup-gcloud/README.md (hunk 1 above)
Comment (averikitsch) [reply setup-gcloud/README.md:66]: done.

File: setup-gcloud/dist/index.js (hunk 2 above)
Comment (averikitsch) [reply setup-gcloud/dist/index.js:37]: done

File: setup-gcloud/src/index.ts (hunk 3 above)
Comment (averikitsch) [reply setup-gcloud/src/index.ts:1]: done.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**